spec/lib/ordering_helper_spec.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+def stub_ordering_helper(helper, eq_model1, eq_model3)
```

Comment:
I think that keyword args will help a lot with the readability of this method -- `stub_ordering_helper(helper:, ordering:, predecessor:, successor:)` where `predecessor` and `successor` are both hashes of form `{ model: eq_model, ordering: 3 }`. 

It's important to be passing the data that your tests depend on into your helper, not to test against arbitrary data that you set away from the test.